### PR TITLE
feat: Restrict extent of `my-map` component to team boundary

### PR DESCRIPTION
**Pull request by DafyddLlyr**
## What does this PR do?
 - Uses `boundary_bbox` on the `PlotNewAddress` component
 - Uses `passport.data["site.boundary.buffered"] on the API to restrict the maps used in `application.html` and `map.html`

### CODE REVIEW

File: api.planx.uk/admin/session/html.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { generateApplicationHTML } from "@opensystemslab/planx-core";
 import { $admin } from "../../client";
-import type { NextFunction, Request, Response } from "express";
+import type { RequestHandler } from "express";
 import type { PlanXExportData } from "@opensystemslab/planx-core/types";
 
+type HTMLExportHandler = RequestHandler<{ sessionId: string }, string>;
```
Comment (DafyddLlyr) [inline api.planx.uk/admin/session/html.ts:6]: This is a neat thing I just learned we can do with Express types

You can do `RequestHandler<PathParams, ResponseBody, RequestBody, QueryParams>`

File: api.planx.uk/send/exportZip.ts [hunk 2]
```diff
@@ -109,17 +109,22 @@ export async function buildSubmissionExportZip({
     }
   }
 
+  const boundingBox = passport.data["property.boundary.site.buffered"];
```
Comment (DafyddLlyr) [inline api.planx.uk/send/exportZip.ts:112]: This is hardcoded on a user-based variable - which I think is realistically fine? 

Alternatively we could hardcode to it a dev-based variable (e.g. `passport.data._somethingWeChose`)

File: editor.planx.uk/src/@planx/components/FindProperty/Public/Map.tsx [hunk 3]
```diff
@@ -128,6 +131,7 @@ export default function PlotNewAddress(props: PlotNewAddressProps): FCReturn {
           showScale
           showNorthArrow
           osProxyEndpoint={`${process.env.REACT_APP_API_URL}/proxy/ordnance-survey`}
+          clipGeojsonData={JSON.stringify(boundaryBBox)}
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/FindProperty/Public/Map.tsx:134]: This is the remaining place where the team's boundary is used - on `PlotNewAddress`

File: api.planx.uk/send/exportZip.ts (hunk 2 above)
Comment (jessicamcinchak) [reply api.planx.uk/send/exportZip.ts:112]: Agree this is realistically fine! Can become a variable we validate existence of on publish in relation to future integrations/schema work :slightly_smiling_face:

**Review by jessicamcinchak — APPROVED** [review]
Looks great, thanks for this last piece! 

I'll go ahead and merge and update testing notes in Trello :slightly_smiling_face:

**Pull request merged**